includes/admin/views/tmpl-order-item.php

```diff
@@ -20,8 +20,9 @@
 <td class="has-row-actions column-primary">
 	<div class="removable">
 		<# if ( true === data.state.isAdding ) { #>
-		<button class="button-link delete">
-			<span class="dashicons dashicons-no"></span>
+		<button class="button-link delete" aria-label="remove item">
```

Comment:
Now that the button has text, the `aria-label` is redundant (it overrides the button text). If it were necessary, it should be added as a translation ready string, but I think in this case, it can be removed.

I think the only thing I am left wondering is whether "Remove Item" is enough, especially since it's _before_ the product name. I wonder if it should be "Remove [Name of Product]" instead--I think what would work would be something like:

```php
<?php printf( __( 'Remove %s', 'easy-digital-downloads' ), '{{{ data.productName }}}' ); ?>
```

because I think the JS is replacing the `{{{}}}` placeholder? This is feeling a little bit like the checkboxes in the post list tables, is why I'm wondering this now, where each checkbox label says, "Select {{{ Hello World! }}}". Using the placeholder in the string gives this button the same kind of effect.

Reply:
I think this will work now.  Is there a difference between `{{{}}}` and `{{}}`?  They seemed to work the same, but I went with `{{{}}}` as suggested.